firmware/library/L2_Utilities/utility_test.cpp

```diff
@@ -0,0 +1,104 @@
+#include "L2_Utilities/utility.hpp"
```

Comment:
Through out this whole file: expected value should be on the left side of the check expression. 

    CHECK(expected == FunctionUnderTest(param1, paramN));